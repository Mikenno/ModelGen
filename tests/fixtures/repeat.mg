print("Test" * 4)
print(4 * "Test")
print(2 * "Test" * 2)
print("Test" * 0)
print("Test" * -1)

print((1, 2, 3) * 4)
print(4 * (1, 2, 3))
print(2 * (1, 2, 3) * 2)
print((1, 2, 3) * 0)
print((1, 2, 3) * -1)