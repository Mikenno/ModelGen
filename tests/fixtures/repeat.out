TestTestTestTest
TestTestTestTest
TestTestTestTest


(1, 2, 3, 1, 2, 3, 1, 2, 3, 1, 2, 3)
(1, 2, 3, 1, 2, 3, 1, 2, 3, 1, 2, 3)
(1, 2, 3, 1, 2, 3, 1, 2, 3, 1, 2, 3)
()
()
